submissions/BlackTowerKing/html-css-popup/css/style.css

```diff
@@ -0,0 +1,229 @@
+/*Start Reset css*/
+
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+a {
+    text-decoration: none;
+}
+
+/* End Reset css*/
+
+/* Start Menu*/
+
+.header {
+    width: 90%;
+    margin: 20px auto;
+}
+
+.top-menu {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.top-menu__item {
+    margin: 0 10px;
+}
+
+.menu-link, .popup-link {
+    font-family: Arial, Helvetica, sans-serif;
+    color: grey;
+    font-size: 1em;
+    text-decoration: none;
+}
+
+.menu-link:hover {
+    text-decoration: underline;
+}
+
+.menu-button {
+    height: 25px;
+    border-radius: 50%;
+    padding: 5px;
+}
+
+.menu-button:hover {
+    background-color: lightgrey;
+}
+
+.profile {
+    height: 40px;
+}
+
+/*End Menu*/
+
+/*Start Popup*/
```

Comment:
It's empty line of your code. I recommend avoid comments like "header", "main" and etc.
Comments need for explaining part of your code, for example I'm understand what's mean 'header' or 'nav' so here it's comments redundant.